metaseq/model_parallel/modules/transformer_layer.py

```diff
@@ -110,7 +112,7 @@ def build_fc2(
         skip_bias_add = self.skip_bias_add
         if full_megatron_init:
             init_method_weights = megatron_utils.scaled_init_method_normal(
-                megatron_init_sigma, num_layers
+                megatron_init_sigma, num_layers * full_megatron_init_scalar
```

Comment:
Nit: It might be more straightforward to scale `megatron_init_sigma` instead?

Reply:
Nah, want to keep that constant since that affects other layers too.

Reply:
The thought here was:
```
init_method_weights = megatron_utils.scaled_init_method_normal(
    megatron_init_sigma * full_megatron_init_scalar, num_layers )
```

Reply:
Ah good callout - that does seem a lot more straightforward; I'll amend!